internal/cache/file/downloader/job.go

```diff
@@ -483,7 +483,7 @@ func (job *Job) validateCRC() (err error) {
 		return
 	}
 
-	crc32Val, err := cacheutil.CalculateFileCRC32(job.fileSpec.Path)
+	crc32Val, err := cacheutil.CalculateFileCRC32(job.cancelCtx, job.fileSpec.Path)
 	if err != nil {
 		return
```

Comment:
cancellation happens during invalidation of job (and cache), so add additional check similar to this https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/internal/cache/file/downloader/job.go#L342 here as well: https://github.com/GoogleCloudPlatform/gcsfuse/blob/ad99e175b1e56ed71ec3d1001782066904d653b3/internal/cache/file/downloader/job.go#L409

Reply:
also, modify this to check Invalid status for completion: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/internal/cache/file/downloader/downloader_test.go#L79